### ISSUE TITLE
More descriptive home page

### ISSUE THREAD

**Issue by girst** [iv-org/invidious#740]
Whenever Invidious gets linked to, people are left confused what it actually is ([latest example](https://old.reddit.com/r/firefox/comments/czy3i5/invidition_youtube_to_invidious_redirector_addon/ez4dw88/)).

I propose adding some informational text at the top of the home page for logged-out users, that explains what Invidious is, some of its main features and provides a call to action (register and import subscriptions). Final text tbd.

Secondly, I'd also switch to showing the *popular* page by default, as it is a better representation of what the instance's registered users actually watch. *Popular* also hosts many familiar channels, letting users know that the content is actually from Youtube, and not hosted by Invidious itself. (videos on the _top_ page aren't representative, as they can appear after only 1 person watched them through Invidious and often are in foreign languages and/or from not widely known personalities)

I've hacked up a quick mock-up in the devtools, but if this is something that is accepted, I can create a proper pull request and put some more effort into a nice looking UI and making it e.g. dismissable.

![screenshot](https://imgur.com/8pZFkhi.png)

<details><summary>code of the mockup (click to expand)</summary>

```html
<!-- inserted instead of the popular/top/trending bar -->
<div>
    <h1>Welcome to Invidious</h1>
    <p>Invidious is a featureful Youtube-player, that respects your privacy.
        </p><p>Features:
        </p><ul>
            <li>Audio-only mode
</li><li>Dark mode
</li><li>No ads
</li><li>Lightweight
</li><li>Doesn't require a Google account
</li><li>Support for Reddit comments
</li><li> ... and others! Click to <a href="https://github.com/omarroth/invidious/blob/master/README.md">learn more</a>
    </li></ul>
        <p><button>create an account and import your subscriptions now</button>
    </p><p>Here is a selection of videos that our community is interested in:
    </p><div class="pure-u-1 pure-u-md-1-4"></div>
</div>
```

</details>

**Comment by girst** [comment]
![screen v2](https://imgur.com/7dZkbbN.png)

still in the proof of concept stage, but I decided to put a gradient behind it, to give it some form and more weight against all the thumbnails. the popular/top/trending bar will reappear when the user hits 'dismiss' or logs in (dismissing it should set a cookie, so it stays hidden).

The big button links to `/login?referer=/data_control`. That page could be a little more user friendly too; I'll tackle that as well (e.g. a two-step instructions box for how to export youtube settings), if that's ok.

The text was tweaked a bit too, although I'm still not 100% happy with it ('featureful', then 'features include'); there are still at least one too many showcased features. There will be a lot of strings added, that will have to be translated.

any feedback on this, @omarroth? (pull request comes in the next few days, once I actually implemented this thing)

**Comment by elypter** [comment]
"dismiss" should be easy to spot as "register" or "log in". there are many privacy focused users who do not want or need an account. pushing users to choices favorable by companies usually indicates an intent to collect data. invidio.us however doesnt have that goal and using their tacticas could drive away some users. nobody should be forced into doing something even if it might be the best option after a thorrough consideration.

**Comment by girst** [comment]
good point on making dismiss more visible. a hidden dismiss button might look like an intentional dark pattern, when in reality the default link colour just has bad contrast. I'll make it a button with a background, so it stands out.

>users who do not want or need an account. 

i'd guess those people would use an addon to redirect youtube links to invidious, not look at the home page all the time. speaking of, that's also something we might want to advertise that it exists.

>usually indicates an intent to collect data.

the latest wording (-> #744) tries to address this by using phrases like "community developed" and "respects your privacy".  how about adding "invidious does not and will never sell or share any data about you or your viewing habits with anyone", or something to that extent? (note that invidious does collect watch history, so writing "we don't collect any data" isn't true)

>using their tacticas could drive away some users. 

i'm toying with the idea of adding a humourous phrase like ["or don't. i'm a button, not a cop"](https://i.imgur.com/mSHi8.jpg). 

i honestly don't think that a call to action will scare off many users; certainly less than are leaving confused due to having no information on what invidious is and how to use it.

**Comment by elypter** [comment]
>"invidious does not and will never sell or share any data about you or your viewing habits with anyone"

you cannot even guarantee it for this instance even though i have no reason to think otherwise and other instances could be used in different ways. a promise should not be part of the source code. being able to select a personal manifesto for the current site could be beneficial though.

**Comment by girst** [comment]
Seperating the welcome message is a good idea. No idea how it could be localized then, though.

**Comment by elypter** [comment]
maybe put it in a separate file in a separate folder inside the config folder

**Comment by comradekingu** [comment]
```
This is a player for YouTube that respects your privacy, and blocks ads.
It also saves bandwidth if you only want the audio.
[+++]

[Create an account] to import subscriptions, or [avoid doing so].
```
Regains valuable screen estate.
It already says "Invidious".
"Welcome" doesn't do much. I argue making it welcoming does.
"Youtube" should be "YouTube".
"Lightweight" should also be apparent.
"Reddit comments" isn't a selling point.

[avoid doing so] minimises the field.
